kibana-reports/server/routes/utils/converters/backendToUi.ts

```diff
@@ -203,6 +203,7 @@ const getUiQueryUrl = (
 ) => {
   const timeFrom = moment(beginTimeMs).toISOString();
   const timeTo = moment(endTimeMs).toISOString();
+
```

Comment:
Was this meant to be included in the PR?

Reply:
fixed